src/machi_cr_client.erl

```diff
@@ -923,10 +927,22 @@ update_proj(#state{proj=undefined}=S) ->
 update_proj(S) ->
     S.
 
-update_proj2(Count, #state{bad_proj=BadProj, proxies_dict=ProxiesDict}=S) ->
+update_proj2(Count, #state{bad_proj=BadProj, proxies_dict=ProxiesDict,
+                           opts=Opts}=S) ->
     Timeout = 2*1000,
     WTimeout = 2*Timeout,
-    Proxies = orddict:to_list(ProxiesDict),
+    SimName = proplists:get_value(simulator_self_name, Opts, cr_client),
+    ExcludedFLUs =
+        case proplists:get_value(use_partition_simulator, Opts, false) of
+            true ->
+                Members = proplists:get_value(simulator_members, Opts, []),
+                {Partitions, _Islands} = machi_partition_simulator:get(Members),
+                [B || {A, B} <- Partitions, A =:= SimName];
```

Comment:
This should be A =:= SimName orelse B =:= SimName, to be affected in either direction.